test/segment-loader.test.js

```diff
@@ -350,23 +365,26 @@ QUnit.test('adjusts the playlist offset if no buffering progress is made', funct
     playlist.uri = 'alternate.m3u8';
     playlist.endList = false;
     loader.playlist(playlist);
-  });
+    this.clock.tick(1);
+  }.bind(this));
   sourceBuffer.buffered = videojs.createTimeRanges([[0, 5]]);
   sourceBuffer.trigger('updateend');
+  this.clock.tick(1);
 
   // the next segment doesn't increase the buffer at all
   assert.equal(this.requests[0].url, '0.ts', 'requested the same segment');
   this.clock.tick(1);
   this.requests[0].response = new Uint8Array(10).buffer;
   this.requests.shift().respond(200, null, '');
   sourceBuffer.trigger('updateend');
+  this.clock.tick(1);
 
   // so the loader should try the next segment
   assert.equal(this.requests[0].url, '1.ts', 'moved ahead a segment');
 
   // verify stats
   assert.equal(loader.mediaBytesTransferred, 20, '20 bytes');
-  assert.equal(loader.mediaTransferDuration, 2, '2 ms (clocks above)');
+  assert.equal(loader.mediaTransferDuration, 1, '1 ms (clocks above)');
```

Comment:
Prior to this change, the request happened immediately. As a result, there were two clock ticks between the request and the response. Now, the request happens later and only one tick happens now (the first tick becomes the tick that triggers the request).